packages/hooks/src/useResizeObserver.mdx

```diff
@@ -22,18 +22,30 @@ import { useResizeObserver } from "@jobber/hooks";
 
 <Playground>
   {() => {
-    const [ref, { width, height }] = useResizeObserver();
-    const title = width > 500 ? "This is big" : "This is small";
+    const [
+      ref,
+      { width, exactWidth, exactHeight, height }
+    ] = useResizeObserver();
     return (
       <div ref={ref}>
-        <Card title={title}>
+        <Card title={`Check out my àçƈéñŦ`} accent={getAccent()}>
           <Content>
             <Text>Width: {width}</Text>
             <Text>Height: {height}</Text>
```

Comment:
What do you think of either taking these out or changing the label to `Width Step` and `Height Step`?